client/lib/post-metadata/index.js

```diff
@@ -136,6 +136,41 @@ PostMetadata = {
 			return [ latitude, longitude ];
 		}
 	},
+
+	/**
+	 * Given a post object, return either "public" or "private", indicating whether
+	 * the geo-location data associated with the post is allowed to be displayed
+	 * publicly.
+	 *
+	 * @param {Object} post Post object
+	 * @returns {string}	Either "public" or "private"
+	 */
+	geoIsSharedPublicly: function( post ) {
+		if ( ! post ) {
+			return 'public';
+		}
+
+		const isSharedPublicly = getValueByKey( post.metadata, 'geo_public' );
+
+		if ( parseInt( isSharedPublicly, 10 ) ) {
+			return 'public';
+		}
+
+		if ( 'undefined' === typeof isSharedPublicly ) {
```

Comment:
You can do a simple `isSharedPublicly === undefined` comparison here. The `typeof` trick is needed only for cases where it's uncertain if the tested variable even exists, like when testing for `window` or `document`.

Reply:
Makes perfect sense.  Thanks!  This is in feb151b.